widget/src/tooltip.rs

```diff
@@ -64,6 +64,12 @@ where
         self
     }
 
+    /// Sets the [`text::Shaping`] strategy of the [`Tooltip`].
+    pub fn text_shaping(mut self, shaping: text::Shaping) -> Self {
```

Comment:
Just `shaping` for consistency?

Reply:
`Radio`, `Toggler`, and `Checkbox` [call it `text_shaping` as well](https://github.com/search?q=repo%3Aiced-rs%2Ficed%20text_shaping&type=code).

Let me know if you prefer to rename it just `shaping` everywhere, but I think that it could be confusing to apply to non-purely-`Text` widgets.

I'm agree with you to call it the simple way on `Text`, but on other widgets maybe being more explicit is better?